component/client/ClientCore.jsx

```diff
@@ -0,0 +1,53 @@
+import React, {Component} from "react";
```

Comment:
I setup the linter to prefer double quotes over single quotes. This is just a personal preference I have.